latest-issue/src/main/res/layout/fragment_latest_issue.xml

```diff
@@ -53,16 +53,16 @@
 
         <androidx.recyclerview.widget.RecyclerView
             android:id="@+id/recycler_view_fragment_latest_issue"
-            android:layout_width="0dp"
-            android:layout_height="0dp"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
             android:clipToPadding="false"
             android:orientation="vertical"
             android:paddingBottom="80dp"
             app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/card_view_fragment_latest_issues_local_issues_message"
+            app:layout_constraintBottom_toBottomOf="@id/swipeRefreshLayout"
+            app:layout_constraintEnd_toEndOf="@id/swipeRefreshLayout"
+            app:layout_constraintStart_toStartOf="@id/swipeRefreshLayout"
+            app:layout_constraintTop_toBottomOf="@+id/swipeRefreshLayout"
```

Comment:
This recyclerview is inside SwipeRefreshLayout so the constraints here will not affect any thing you need to back to old constraints and move the constraints to `SwipeRefreshLayout` not to recyclerview